Gemfile

```diff
@@ -14,15 +14,26 @@ gem "dry-auto_inject"
 gem "dry-matcher"
 gem "dry-validation"
 
-if ENV['USE_LOCAL_GEMS']
+case ENV['GEMS_SOURCE']
+when 'local'
   gem "reform", path: "../reform"
   gem "reform-rails", path: "../reform-rails"
   gem "trailblazer-operation", path: "../trailblazer-operation"
   gem "trailblazer-macro", path: "../trailblazer-macro"
   gem "trailblazer-macro-contract", path: "../trailblazer-macro-contract"
   gem "trailblazer-activity", path: "../trailblazer-activity"
   gem "trailblazer-context", path: "../trailblazer-context"
-else
+when 'github'
+  gem "reform", github: "trailblazer/reform"
+  gem "reform-rails", github: "trailblazer/reform-rails"
+  gem "trailblazer-operation", github: "trailblazer/trailblazer-operation"
+  gem "trailblazer-macro", github: "trailblazer/trailblazer-macro"
+  gem "trailblazer-macro-contract", github: "trailblazer/trailblazer-macro-contract"
+  gem "trailblazer-activity", github: "trailblazer/trailblazer-activity"
+  gem "trailblazer-context", github: "trailblazer/trailblazer-context"
+when 'custom'
+  eval_gemfile('GemfileCustom')
```

Comment:
how if give a warn if the file is not exist?
or we should update https://github.com/trailblazer/trailblazer/blob/master/CONTRIBUTING.md about this feature

Reply:
We could add a comment.